build.ps1

```diff
@@ -565,6 +596,7 @@ switch -Exact ($Target) {
         #VSX v110
 		VSX v120
         VSX v140
+		VSX v141
```

Comment:
I can comment this one if thats what you prefer. Probably not that many people have adopted VS2017 yet.